src/content/docs/how-to-work-on-quizzes.mdx

```diff
@@ -19,12 +19,26 @@ Every certification will have a series of quizzes to test camper's knowledge as
 
 ## Basic Structure for Quizzes
 
-Each quiz will have a total of 20 questions. Every quiz question will have a set of 4 options (1 correct answer and 3 distractors).
+Each quiz can have either a total of 10 or 20 questions. Every quiz question will have a set of 4 options (1 correct answer and 3 distractors).
 
-To pass the quiz, a camper will need to get at least 17 questions correct. If they fail the quiz, then there will be a brief cool down period before they are able to take the quiz again.
+To pass the quiz, a camper will need to get at least 90% of the questions correct. If they fail the quiz, then there will be a brief cool down period before they are able to take the quiz again.
```

Comment:
I missed this in the last review. We decided against cool down periods for quizzes. Only exams will have cool down periods. 

```suggestion
To pass the quiz, a camper will need to get at least 90% of the questions correct.
```